src/tools/dart-devtools.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Dart DevTools
+description: A suite of performance tools for Dart and Flutter.
+toc: false
+---
+
+Dart DevTools is a suite of performance tools for Dart and Flutter.
+DevTools supports source-level debugging of Dart command-line apps,
+as well as viewing general log and diagnostics information about a running app.
+
+We expect developers to use DevTools as part of their existing
```

Comment:
It's not quite clear to me what this sentence adds. Should I read is as saying that DevTools works for both IDE and command-line based developer workflows?

Reply:
I took it from the ddt page, and understood it as meaning "this isn't an IDE". But your wording is better.